queries/synths/useBaseFeeRateQuery.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from 'react-query';
+import { useRecoilValue } from 'recoil';
+import Connector from 'containers/Connector';
+import Wei from '@synthetixio/wei';
+import { ethers } from 'ethers';
+
+import { CurrencyKey } from 'constants/currency';
+import { appReadyState } from 'store/app';
+
+const useBaseFeeRateQuery = (currencyKey: CurrencyKey | null) => {
+	const isAppReady = useRecoilValue(appReadyState);
+	const { synthetixjs } = Connector.useContainer();
+
+	return useQuery(
+		['synths', 'baseFeeRate'],
```

Comment:
Can you add this to `constants/queryKeys.ts` and pass in the currencyKey? Otherwise the cache may not update when you query for a different synth.

Example: https://github.com/Kwenta/kwenta/blob/f7d4ea9aa2312197c2a16e41a262d3cc15ccb885/queries/rates/usePeriodStartSynthRateQuery.ts#L24

Reply:
Sure. I will do it.